### PR TITLE
Adds standalone view with simple CSS for developer agreement TOS, useful for iframing.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @ngokevin

### CODE REVIEW

File: mkt/developers/templates/developers/terms_standalone.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<html>
+  <head>
+    {{ css('mkt/devreg') }}
+    <style type="text/css">
```
Comment (chuckharmston) [inline mkt/developers/templates/developers/terms_standalone.html]: This is sort of lame, but some of the CSS is very specific to devhub. This is just enough for a nice, iframe-able presentation:

screen shot 2015-09-21 at 12 16 24 pm

File: mkt/developers/templates/developers/terms_standalone.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<html>
+  <head>
+    {{ css('mkt/devreg') }}
```
Comment (ngokevin) [inline mkt/developers/templates/developers/terms_standalone.html]: this seems more CSS than we need, do we actually need this?

Comment (chuckharmston) [inline mkt/developers/templates/developers/terms_standalone.html]: It's a pretty simple way to add all the fonts and resets. I was shooting for simplicity over optimization, given the very specific nature of it.

File: mkt/account/tests/test_serializers.py [hunk 3]
```diff
@@ -86,25 +86,14 @@ class TestTOSSerializer(mkt.site.tests.TestCase):
     def setUp(self):
         self.account = UserProfile()
 
-    def serializer(self, lang='pt-BR'):
```
Comment (ngokevin) [inline mkt/account/tests/test_serializers.py]: are we supporting en-gb yet??

**Comment by ngokevin** [comment]
r+wc

File: mkt/account/tests/test_serializers.py (hunk 3 above)
Comment (chuckharmston) [inline mkt/account/tests/test_serializers.py]: Yes, but this is in a test.

Comment (chuckharmston) [inline mkt/account/tests/test_serializers.py]: That is no longer necessary, since the standalone TOS view is locale-aware.

**Comment by chuckharmston** [comment]
@ngokevin Which changes?

**Comment by chuckharmston** [comment]
Just pulling in reset, base, and typography now. Should be about the minimum amount of CSS required.

**Pull request merged**